packages/lib/models/Note.test.ts

```diff
@@ -332,4 +332,39 @@ describe('models_Note', function() {
 		expect(sortedNotes3[4].id).toBe(note2.id);
 	}));
 
+	it('should copy the note to target folder and cancel conflict', (async () => {
+		const folder1 = await Folder.save({ title: 'Folder' });
+		const conflictFolder = await Folder.save({ title: Folder.conflictFolderTitle(), id: Folder.conflictFolderId() });
+		const note1 = await Note.save({ title: 'note', parent_id: conflictFolder.id, is_conflict: 1, conflict_parent_id: folder1.id });
+		const note2 = await Note.copyToFolder(note1.id, folder1.id);
+
+		// Targed should be copied and have conflict removed.
+		expect(note2.id == note1.id).toBe(false); // ID must be different.
```

Comment:
Always use strict equality ===. I can't add an eslint rule for it because there would be too much legacy code to convert, but all new code should use strict equality.